scripts/screens/WarriorDenScreen.py

```diff
@@ -71,7 +71,7 @@ def handle_event(self, event):
                         if "mediator" in description and self.save_button.is_enabled:
                             # only create the mediator list if needed to check
                             mediator_list = list(filter(
-                                lambda x: x.status == "mediator" and not x.dead and not x.outside, Cat.all_cats_list
+                                lambda x: x.status.is_mediator_any() and not x.dead and not x.outside, Cat.all_cats_list
```

Comment:
why `is_mediator_any` instead of `is_mediator`?

Reply:
Ah. The original seems to be an oversight: in `medicine cat` below, both medicine cats and their apprentices are counted in the enable/disable toggle, but here only mediators are. I think this might've just been a mistake, as in most cases we do want both (and annoyingly it autofills with "any" versus staying blank) but looking at it I'm not sure whether I should actually change it back. Thoughts?

Reply:
Optimally we want to _only_ refactor and fix things afterwards. Maybe revert this back to the "original logic" and fix in a separate PR with dedicated testing?

Reply:
Makes sense to me! I'll revert and add a note to self.